docs/INSTALL.md

```diff
@@ -32,7 +31,7 @@ To install everything (client and server), install these packages:
 
 (to build `libsodium` after):
 
-    git clone git@github.com:jedisct1/libsodium.git
```

Comment:
this doesn't work
so changed it

Reply:
Yes, HTTPS is safer option for read-only access, for git@github.com to work, some additional configuration is needed.